src/sections/Faq/index.tsx

```diff
@@ -1,7 +1,152 @@
+    questions: [
+      {
+        question: "How do I get to Hack the 6ix?",
+        answer:
+          "The venue is still being decided, more updates to come on this!",
+      },
+      {
+        question: "What should I bring?",
+        answer:
+          "Make sure to bring your laptop (or desktop) and a piece of valid student ID or government ID! You can also bring a pillow and blanket if you want to get comfy. Everything else will be provided for you!",
+      },
+      {
+        question: "Will there be hardware provided at the event?",
+        answer:
+          "We have a variety of hardware that can be borrowed at our hardware station for free, including Raspberry Pi’s, Arduinos, sensors, and breadboards. Due to limited quantity, hardware will be lent out on a first come, first serve basis. If you are unsure whether or not we have a certain piece of hardware that you will need for your hack, bring your own to the event!",
+      },
+      {
+        question: "How much does it cost to attend?",
+        answer:
+          "Absolutely nothing! Hack the 6ix is a completely free event run by a non-profit organization. All food, resources, and accommodations for hacking for the entire event will be provided free of charge.",
+      },
+    ],
+  },
+  {
+    category: "Health & Safety",
+    questions: [
+      {
+        question: "What COVID-19 precautions will be in place at the event?",
+        answer:
+          "We will be following the guidance put forth by the Ontario Government for large events at the time of the event. More details will follow closer to the event and any major changes to guidelines or precautions will be communicated to event participants.",
+      },
+    ],
+  },
+];
 
 function Faq() {
-  return <PageSection>FAQ</PageSection>;
+  return (
+    <PageSection>
+      <Typography textType="heading2" textColor="primary-3" as="h2">
+        Frequently Asked Questions
+      </Typography>
+      <div className={faqs}>
+        {questionGroups.map(({ category, questions }) => (
+          <div className={faqGroup}>
+            <div className={faqCategoryTitle}>
+              <Typography
+                className={faqCategoryTitleHeading}
+                textType="heading3"
+                textColor="primary-3"
+                as="h3"
+              >
+                {category}
+              </Typography>
+            </div>
+            {questions.map(({ question, answer }) => (
+              <Accordion
+                title={
+                  <Typography textType="heading4" textColor="copy-dark" as="h4">
+                    {question}
+                  </Typography>
+                }
+              >
+                <Typography
+                  className={faqAnswer}
+                  textType="paragraph1"
+                  textColor="copy-dark"
+                  as="p"
+                >
+                  {answer}
+                </Typography>
+              </Accordion>
+            ))}
```

Comment:
This makes more sense in markup as an `ul` or even better a `dl`

Reply:
Latest commit uses `as="dd/dt"` in `Faq/index.tsx`, would you like me to make that the default accordion behavior?